tap_snowflake/client.py

```diff
@@ -95,21 +94,21 @@ class TableProfile:
 class SnowflakeConnector(SQLConnector):
     """Connects to the Snowflake SQL source."""
 
-    def get_private_key(self):
+    def get_private_key(self) -> bytes:
         """Get private key from the right location."""
         try:
             encoded_passphrase = self.config["private_key_passphrase"].encode()
         except KeyError:
             encoded_passphrase = None
 
         if "private_key_path" in self.config:
-            with open(self.config["private_key_path"], "rb") as key:
+            with Path.open(self.config["private_key_path"], "rb") as key:
```

Comment:
You're right & You're quick on the review :). 

So far have only made the changes mechanically and have not run pytest yet.

Reply:
Actually, this error might also be present in target-snowflake. I'll check & submit a fix if needed.
@edgarrmondragon . target-snowflake issues have greater potential to impact end users (since there are many more of them)